tests/src/test/scala/slinky/core/annotations/LocalImportsComponent.scala

```diff
@@ -1,11 +1,11 @@
 package slinky.core.annotations
 
-// compile-only test, this used to crash the macro annotation
-object LocalImportsComponent {
-  import slinky.core.StatelessComponent
+// // compile-only test, this used to crash the macro annotation
```

Comment:
Could we move this (and the other commented out tests) to a `scala-2` source folder until we can handle `@react` properly?